examples/cfrl_mnist.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>Emphasize at the start that this example is using a <code>tensorflow</code> autoencoder and backend and <code>pytorch</code> is also supported.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>